portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,34 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> list;
```

Comment:
Please add some documentation to this field. `list` is too generic and the lack of documentation makes it hard to figure out what we're expected to see in this list.

Reply:
The list was left generic because in the future it will be replaced by user-inputted comments. Still renamed for better understanding for now.